lib/windows/app/actions/deviceActions.js

```diff
@@ -165,7 +165,8 @@ function logDeviceListerError(error) {
         // to a libusb-compatible-driver will fail to enumerate and trigger a
         // LIBUSB_* error. This is the case of a nRF52840 in DFU mode.
         // We don't want to show an error to the user in this particular case.
-        if (error.message === 'LIBUSB_ERROR_NOT_FOUND' && error.usb.deviceDescriptor) {
+        // errorCode 105 = LIBUSB_ERROR_NOT_FOUND
+        if (error.errorCode === 105 && error.usb.deviceDescriptor) {
```

Comment:
@mathiasap could the numeric value instead be replaced with a reference DeviceLister.ErrorCodes.LIBUSB_ERROR_NOT_FOUND?